spec/lib/submission_builder/ty2024/states/md/documents/md502_spec.rb

```diff
@@ -446,5 +446,67 @@
         expect(xml.at("Form502 TaxWithheld")&.text).to eq('500')
       end
     end
+
+    context "AuthToDirectDepositInd" do
+      before do
+        intake.bank_authorization_confirmed = "yes"
```

Comment:
sorry if this is a ridiculous question but does this...work? i would usually
`intake.update(bank_authorization_confirmed: "yes")`

Reply:
yeahh i'm not sure what's going on but
`allow(intake).to receive(:bank_authorization_confirmed_yes?).and_return true` and `intake.bank_authorization_confirmed = "yes"`

passes this spec, but fails with:
`intake.bank_authorization_confirmed_yes!`
`intake.update(bank_authorization_confirmed: "yes")`